code/controllers/subsystem/job.dm

```diff
@@ -665,6 +672,13 @@ SUBSYSTEM_DEF(job)
 		message_admins(msg)
 		CRASH(msg)
 
+///Lands specified mob at a random spot in the hallways
+/datum/controller/subsystem/job/proc/DropLandAtRandomHallwayPoint(mob/living/living_mob)
+	var/turf/spawn_turf = get_safe_random_station_turf(list(typesof(/area/hallway)))
```

Comment:
isn't typesof already a list? why double wrap it?

Reply:
good point